config/katello.migrations/20250317000666_remove_ovirt.rb

```diff
@@ -0,0 +1 @@
+answers.delete('foreman::compute::ovirt')
```

Comment:
We should also uninstall the rpms, right? Or will this be taken care of by the packaging?

Reply:
No, this only stops managing it. That means if it was installed, it remains installed. If it wasn't installed, it'll remain that way too. Packaging is the best place to take care of removal.